src/dos/drive_virtual.cpp

```diff
@@ -22,41 +22,158 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <fstream>
 
 #include "dos_inc.h"
 #include "support.h"
+#include "shell.h"
 #include "cross.h"
 
+#define default_date DOS_PackDate(2002, 10, 1)
+#define default_time DOS_PackTime(12, 34, 56)
+
```

Comment:
If you can rebase against the latest `main`, I pushed a tiny change to let the DOS time functions be evaluated compile-time, so these two can both become:
``` c++
constexpr default_date DOS_PackDate(2002, 10, 1);
constexpr default_time DOS_PackTime(12, 34, 56);
```
(Zero issue with the code! Just a small improvement we can start to use)

Reply:
Thanks for the code fix. Changed from #define to constexpr accordingly.